sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -218,6 +222,7 @@ export class KeyClient {
     createRsaKey(name: string, options?: CreateRsaKeyOptions): Promise<KeyVaultKey>;
     getDeletedKey(name: string, options?: GetDeletedKeyOptions): Promise<DeletedKey>;
     getKey(name: string, options?: GetKeyOptions): Promise<KeyVaultKey>;
+    getRandomBytes(count: number, options?: GetRandomBytesOptions): Promise<Uint8Array>;
```

Comment:
The swagger has this modelled as returning an object with an optional value - if we modeled that here it would return a `Promise<Uint8Array>`

How do folks feel about this instead? I tested the error conditions and they all throw with a useful error. Under what conditions would this return undefined but not reject the promise?

Reply:
I'd like to know under what circumstances the service would return `undefined`. Unless it has some special meaning (which case, and I think those would be covered by errors), I'd rather we return an empty Uint8Array if we saw undefined.

Reply:
Thanks for following up! I spoke to service folks and it should always return a value on success - so I will update the swagger

Reply:
Opened to reflect this https://github.com/Azure/azure-rest-api-specs/pull/14982